app/components/add-layer/template.hbs

```diff
@@ -1,19 +1,25 @@
 {{#liquid-if isOpen use='toDown'}}
     <div class="add-layer {{if isOpen 'open'}}">
         <div class="container">
+          
             <div class="row p-v-md">
-                <div class="col-sm-2">
-                    <button class="btn btn-default btn-block" {{action 'addLayer' 'wiki'}}>{{fa-icon "wikipedia-w"}} <br /> Wiki</button>
+                
+                <div class="col-xs-2">
+                    <button class="btn btn-default btn-block" {{action 'addLayer' 'wiki'}}>{{fa-icon "wikipedia-w"}} <br /> Add Wiki layer </button>
                 </div>
-                <div class="col-sm-2">
-                    <button class="btn btn-default btn-block" {{action 'addLayer' 'link'}}>{{fa-icon "link"}} <br /> Link</button>
+                <div class="col-xs-2">
+                    <button class="btn btn-default btn-block" {{action 'addLayer' 'link'}}>{{fa-icon "link"}} <br /> Add Link layer</button>
                 </div>
-                <div class="col-sm-2">
-                    <button class="btn btn-default btn-block" {{action 'addLayer' 'file'}}>{{fa-icon "download"}} <br /> Download</button>
+                <div class="col-xs-2">
+                    <button class="btn btn-default btn-block" {{action 'addLayer' 'file'}}>{{fa-icon "download"}} <br /> Add Download layer</button>
                 </div>
+
                 <div class="col-sm-2">
                     <button class="btn btn-default btn-block" {{action 'addLayer' 'image'}}>{{fa-icon "image"}} <br /> Background Image</button>
                 </div>
+                <div class="col-xs-2">
+                    <button class="btn btn-default btn-block" {{action 'addLayer' 'advanced'}}>{{fa-icon "paint-brush"}} <br /> Add Advanced layer</button>
+                </div>
```

Comment:
'Advanced' layer is fine for internal use but I think we should brainstorm another name for it in the UI